frontend/src/containers/ImageContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react';
+
+import ImageComponent from '../components/ImageComponent';
+import { IMAGE_API } from '../data/ApiConstants';
+import Image from '../models/Image';
+
+interface Props {}
+
+interface State {
+  images: Array<Image>;
+}
+
+class ImageContainer extends PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = { images: [] };
+  }
+
+  componentDidMount() {
+    fetch(IMAGE_API, { mode: 'cors' })
+      .then((res) => res.json())
```

Comment:
tiny nit: I would move this into a function that returns a Promise (maybe in `src/data/something.ts`) so that components don't need to care about the fetch internals and only handle the responses